tests/DocumentationParserTest.php

```diff
@@ -325,21 +320,21 @@ public function testApiLinks()
         $page_version = $this->page->getVersion();
 
         // expected url format resulting from rewriting api shortcode links
-        $url_format = '<a href="http://api.silverstripe.org/search/lookup/?q=%s&version='.$page_version.'&module=documentationparsertest">%s</a>';
+        $html_format = '&nbsp;<a href="http://api.silverstripe.org/search/lookup/?q=%s&version='.$page_version.'&module=documentationparsertest">%s</a>';
```

Comment:
It wasn't "needed" before but neither was it working correctly before!

Anyhow, I think I've finally sorted the gobble problem :) Bloody regexes are tricky to get exactly right...Ready to merge.